web/src/components/Dollars.test.js

```diff
@@ -29,19 +29,7 @@ describe('Dollars formatting container', () => {
     });
   });
 
-  test('prints full numbers if long prop is set', () => {
+  test('prints dollar-formatted numbers', () => {
     expect(render(<Dollars long>2857298672</Dollars>)).toMatchSnapshot();
```

Comment:
@mgwalker should the "long" be removed on line 33?

Reply:
Oops. Fixed!